api.md

```diff
@@ -613,13 +615,17 @@ Drop all chunks older than 3 months ago:
 SELECT drop_chunks(interval '3 months');
 ```
 
-The expected output:
+An example output: 
```

Comment:
I would drop the `An` here and just name it `Example output:`

Reply:
I agree